src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -1614,29 +1614,17 @@ public void showEntry(final BibEntry be) {
 
         }
 
-        EntryEditor entryEditor;
-        int divLoc = -1;
         String visName = null;
         if ((getShowing() != null) && isShowingEditor()) {
             visName = ((EntryEditor) splitPane.getBottomComponent()).getVisiblePanelName();
         }
-        if (getShowing() != null) {
-            divLoc = splitPane.getDividerLocation();
-        }
 
         // We must instantiate a new editor.
-        entryEditor = new EntryEditor(frame, BasePanel.this, be);
+        EntryEditor entryEditor = new EntryEditor(frame, BasePanel.this, be);
         if (visName != null) {
             entryEditor.setVisiblePanel(visName);
         }
-        splitPane.setBottomComponent(entryEditor);
-
-        if (divLoc > 0) {
```

Comment:
The height of the editor (or preview) is handled by the `BasePanel#adjustSplitter()` method which just sets the saved height.

The height of the preview panel is saved in `BasePanel#saveDividerLocation()` without any maximum/minimum size adjustment. Currently the minimum size of the preview panel is 5px.
 
The entry editor height is saved in 3 places without any adjustments, but the minimum height of the editor is limited via the editor toolbar and the layout of the tabs (which depends on the entry type) .